src/types/constants/actions/apply.ts

```diff
@@ -1,36 +1,36 @@
-export const UPDATE_APPLICATION_EMAIL = 'UPDATE_APPLICATION_EMAIL';
-export type UPDATE_APPLICATION_EMAIL = typeof UPDATE_APPLICATION_EMAIL;
+export const UPDATE_APPLY_EMAIL_VALUE = 'UPDATE_APPLICATION_EMAIL';
+export type UPDATE_APPLY_EMAIL = typeof UPDATE_APPLY_EMAIL_VALUE;
 
-export const UPDATE_APPLICATION_FIRST_NAME = 'UPDATE_APPLICATION_FIRST_NAME';
-export type UPDATE_APPLICATION_FIRST_NAME = typeof UPDATE_APPLICATION_FIRST_NAME;
+export const UPDATE_APPLY_FIRST_NAME_VALUE = 'UPDATE_APPLICATION_FIRST_NAME';
+export type UPDATE_APPLY_FIRST_NAME = typeof UPDATE_APPLY_FIRST_NAME_VALUE;
 
-export const UPDATE_APPLICATION_LAST_NAME = 'UPDATE_APPLICATION_LAST_NAME';
-export type UPDATE_APPLICATION_LAST_NAME = typeof UPDATE_APPLICATION_LAST_NAME;
+export const UPDATE_APPLY_LAST_NAME_VALUE = 'UPDATE_APPLICATION_LAST_NAME';
+export type UPDATE_APPLY_LAST_NAME = typeof UPDATE_APPLY_LAST_NAME_VALUE;
 
-export const UPDATE_APPLICATION_ORGANIZATION = 'UPDATE_APPLICATION_ORGANIZATION';
-export type UPDATE_APPLICATION_ORGANIZATION = typeof UPDATE_APPLICATION_ORGANIZATION;
+export const UPDATE_APPLY_ORGANIZATION_VALUE = 'UPDATE_APPLICATION_ORGANIZATION';
+export type UPDATE_APPLY_ORGANIZATION = typeof UPDATE_APPLY_ORGANIZATION_VALUE;
 
-export const UPDATE_APPLICATION_DESCRIPTION = 'UPDATE_APPLICATION_DESCRIPTION';
-export type UPDATE_APPLICATION_DESCRIPTION = typeof UPDATE_APPLICATION_DESCRIPTION;
+export const UPDATE_APPLY_DESCRIPTION_VALUE = 'UPDATE_APPLICATION_DESCRIPTION';
+export type UPDATE_APPLY_DESCRIPTION = typeof UPDATE_APPLY_DESCRIPTION_VALUE;
 
-export const UPDATE_APPLY_OAUTH_APP_TYPE =
+export const UPDATE_APPLY_OAUTH_APP_TYPE_VAL =
```

Comment:
Seems long, but I think it would be better to not break the naming convention here
```suggestion
export const UPDATE_APPLY_OAUTH_APP_TYPE_VALUE =
```

Reply:
yeah... i set a max identifier length of 32, so this breaks that rule if i add the "UE". so here are the options i can think of:

1. disable the max length rule with a comment here
2. rename it to something that spells out "VALUE" but removes "OAUTH", like `UPDATE_APPLY_APP_TYPE_VALUE` or `UPDATE_APPLY_CLIENT_TYPE_VALUE`. did this with the redirect URI; makes the OAuth part implied but does match the pattern.
3. leave as is

thoughts? will go with whichever you prefer.

Reply:
I like option 2, but let's just leave it as is for now. 👍